libraries/AP_NavEKF3/AP_NavEKF3_Control.cpp

```diff
@@ -88,11 +88,9 @@ void NavEKF3_core::setWindMagStateLearningMode()
             }
 
             // set the wind state variances to the measurement uncertainty
-            for (uint8_t index=22; index<=23; index++) {
```

Comment:
great find! 
I've looked through the rest of the calls to zeroCols() and don't find any others in a loop like this in EK3 or EK2